torch/cuda/memory.py

```diff
@@ -1038,4 +1044,5 @@ def use_mem_pool(pool: MemPool, device: Union[Device, int] = None):
         yield
     finally:
         _cuda_endAllocateCurrentStreamToPool(device_index, pool.id)
+        _cuda_releasePool(device_index, pool.id)
```

Comment:
Therefore, _cuda_endAllocateCurrentStreamToPool no longer decrefs, check all use sites

Reply:
This is only use site, confirmed